test-framework/src/test/java/api/app/astrodao/com/tests/NotificationsApiTests.java

```diff
@@ -0,0 +1,155 @@
+        notificationsApiSteps.assertCollectionElementsHasValue(notifications.getData(), r -> r.getMetadata() != null, "data/metadata");
+        //TODO: Add verification that sorting is done correctly
+    }
+
+    @Test
+    @Severity(SeverityLevel.CRITICAL)
+    @Story("User should be able to get list of notifications with query param: [sort, fields]")
+    void getListOfNotificationsWithSortFieldsParams() {
+        int count = 50;
+        int page = 1;
+        Map<String, Object> query = Map.of(
+                "sort", "id,DESC",
+                "fields", "createdAt,id,metadata"
+        );
+        ResponseEntity<String> response = notificationsApiSteps.getNotifications(query);
+        notificationsApiSteps.assertResponseStatusCode(response, HttpStatus.OK);
+
+        NotificationsResponse notifications = notificationsApiSteps.getResponseDto(response, NotificationsResponse.class);
+
+        notificationsApiSteps.assertDtoValueGreaterThan(notifications, r -> r.getTotal().intValue(), count, "total");
+        notificationsApiSteps.assertDtoValueGreaterThan(notifications, r -> r.getPageCount().intValue(), 1, "pageCount");
+        notificationsApiSteps.assertDtoValue(notifications, r -> r.getPage().intValue(), page, "page");
+        notificationsApiSteps.assertDtoValue(notifications, r -> r.getCount().intValue(), count, "limit");
+        notificationsApiSteps.assertCollectionHasCorrectSize(notifications.getData(), count);
+        notificationsApiSteps.assertCollectionElementsHasValue(notifications.getData(), r -> r.getCreatedAt() != null, "data/createdAt");
+        notificationsApiSteps.assertCollectionElementsHasValue(notifications.getData(), r -> r.getId() != null, "data/id");
+        notificationsApiSteps.assertCollectionElementsHasValue(notifications.getData(), r -> r.getMetadata() != null, "data/metadata");
+        //TODO: Ask a question why we get dao field in response
+        notificationsApiSteps.assertCollectionElementsHasValue(notifications.getData(), r -> r.getDao() != null, "data/dao");
+
+        notificationsApiSteps.assertCollectionElementsHasNoValue(notifications.getData(), r -> r.getDaoId() == null, "data/daoId");
+        notificationsApiSteps.assertCollectionElementsHasNoValue(notifications.getData(), r -> r.getTargetId() == null, "data/targetId");
+        notificationsApiSteps.assertCollectionElementsHasNoValue(notifications.getData(), r -> r.getSignerId() == null, "data/signerId");
+        notificationsApiSteps.assertCollectionElementsHasNoValue(notifications.getData(), r -> r.getType() == null, "data/type");
+        notificationsApiSteps.assertCollectionElementsHasNoValue(notifications.getData(), r -> r.getStatus() == null, "data/status");
+    }
+
+    @Test
+    @Severity(SeverityLevel.CRITICAL)
+    @Story("User should be able to get list of notifications with query param: [sort, s]")
```

Comment:
Please add Description tag with a description of what param 's' means. In my opinion it will add more details and context to what does it mean and what the test verifies. Because currently I have no idea what is that